openhexa/sdk/pipelines/parameter.py

```diff
@@ -358,31 +359,42 @@ def validate(self, value: typing.Optional[typing.Any]) -> Dataset:
 }
 
 
+class ParameterWidget(StrEnum):
```

Comment:
Could you add a description on where the developer could find this list in the graphql doc (like name of enum)